src/components/pages/StudentsPage/index.tsx

```diff
@@ -58,7 +65,12 @@ export default function StudentsPage({ classroomName }: ClassroomProps) {
   return (
     <main>
       <Typography variant='h4' sx={{ color: 'white', mb: 4 }}>
-        Hello {name}! Welcome to your classroom: {classroomName}
+        {!removedFromClass && (
+          <>
+            Hello {name}! Welcome to your classroom: {classroomName}
+          </>
+        )}
+        {removedFromClass && <>You have been removed from the classroom.</>}
```

Comment:
Convert to a terniary operator. Also "your teacher has removed you..."
```suggestion
        {removedFromClass ? 
          (Your teacher has removed you from the classroom) :
          (Hello {name}! Welcome to your classroom: {classroomName})
         }
```